app/src/organisms/ODD/CameraSettings/index.tsx

```diff
@@ -34,6 +34,7 @@ export interface CameraSettingsProps {
   not general settings context. */
   storageInfo: RobotStorageInfo | null
   isCameraRequired: boolean | null
+  runId?: string
```

Comment:
```suggestion
  runId: string | null
```
While we could use an optional prop to achieve the desired result, I think we want consumers of this component to think about whether or not they need to pass a `runId` or not, because whether or not someone does pass in a `runId` results in behavior that is either a bug (or not a bug).

We don't do a great job of modeling this pattern in our app codebase, but a pattern we want to encourage moving forward is roughly the following heuristic: if a user fails to pass in a prop when they should, does that entirely break underlying component behavior or only result in a minor UI/stylistic difference? If it breaks underlying functionality, it shouldn't be an optional prop. Side note: In practice, most of the places where the app currently passes props as optional should actually be `null`.

A good example of a prop that should be optional is something like a `highlight?: boolean` prop, in which failure to pass it in means we don't highlight a labware with a blue border or something. In this instance, the component isn't fundamentally broken because the dev didn't pass in `highlight: true` when they should have done so.